gdcdictionary/schemas/genomic_profile_harmonization_workflow.yaml

```diff
@@ -0,0 +1,50 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "genomic_profile_harmonization_workflow"
+title: Genomic Profile Harmonization Workflow
+type: object
+namespace:  http://gdc.nci.nih.gov
+category: analysis 
+program: '*'
+project: '*'
+description: >
+  Metadata for the harmonization of FoundationOne Genomic Profiling reports.
+additionalProperties: false
+submittable: false
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+  - state
+
+links:
+  - name: submitted_genomic_profiles
+    backref: genomic_profile_harmonization_workflows
+    label: performed_on
+    target_type: submitted_genomic_profile
+    multiplicity: one_to_one
+    required: true
+
+required:
+  - submitter_id
+  - workflow_link
```

Comment:
Workflow link refers to the link to the open source code used to run the workflow. So these are the github commits in our harmonization pipelines. Will that not be the case here?